Runtime/Antlr3.Runtime/CommonToken.cs

```diff
@@ -250,11 +250,13 @@ public override string ToString()
             return "[@" + TokenIndex + "," + start + ":" + stop + "='" + txt + "',<" + type + ">" + channelStr + "," + line + ":" + CharPositionInLine + "]";
         }
 
+#if !PORTABLE
```

Comment:
:memo: The PCL library can be used by an application running on a Windows desktop (full framework). In this case, excluding this code would result in an undesirable behavior change relative to users who shipped the net40-client build with their application.